### PR TITLE
2024-lifelock

### PR DESCRIPTION
**Pull request by fabiandudler**
Add article on Lifelocks

### CODE REVIEW

**Comment by carinaschrenk** [comment]
# Internal Feedback Round 3A 05.12 

#### Thoughts
- The graphic is more fitting to deadlocks and not lifelocks 
  * if the article is supposed to complement the one on deadlocks the graphic needs to reflect something that is moving but still stuck 
- A bit too wordy 

#### Ideas 
- Adopt graphic as said above

**Comment by evos96** [comment]
External feedback:

Part of participants that are computer science students/graduates: 62%

Part of participants that would read the booklet if it was not for this survey: 50 %

Clarity of the article:
Very clear: 50 %
Clear: 25 %
Not clear: 25 %

Technical vs. Non-technical:
Good balance between technical depth and readability: 62 %
The article was too technical: 38 %

Engagement:
I stayed engaged throughout: 87 %
I lost interest midway: 13 %

Structure of the article:
The structure is clear and easy to follow: 87 %
The structure was confusing: 13 %

Part of participants that understand the purpose of the article: 75 %

Part of participants that think the drawing helps to understand the meaning of the article:  87 %

Factual errors/inaccuracies or additional comments:
-
